src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -689,15 +663,8 @@ void DoorLockServer::ClearCredentialCommandHandler(
 {
     emberAfDoorLockClusterPrintln("[ClearCredential] Incoming command [endpointId=%d]", commandPath.mEndpointId);
 
-    auto modifier = getFabricIndex(commandObj);
-    if (kUndefinedFabricIndex == modifier)
-    {
-        emberAfSendImmediateDefaultResponse(EMBER_ZCL_STATUS_FAILURE);
-        return;
-    }
-
-    auto sourceNodeId = getNodeId(commandObj);
-    if (chip::kUndefinedNodeId == sourceNodeId)
+    auto operationalId = getOperationalId(commandObj);
+    if (operationalId == OperationalId::UndefinedOperationalId())
```

Comment:
same here: need a 'isValid' that  checks that both nodeid and fabricid are valid.
I see more of the same below, will not comment on each one.